style.css

```diff
@@ -1,11 +1,16 @@
+@import url('https://fonts.googleapis.com/css2?family=Roboto');

```

Comment:
I have added the fonts Roboto and Open sans to trial, they appear to be working pretty good in my code and they are good in terms of readability and aesthetic effect.